dist/performance.js

```diff
@@ -264,7 +264,7 @@ function getLcpPreloadInfo(rawDoc, lcpUrl) {
 
         let src = link.href;
         if (link.hasAttribute('imagesrcset')) {
-            src = splitSrcSet(link.imagesrcset).find(src => src == lcpUrl);
+            src = splitSrcSet(link.imagesrcset || link.getAttribute('imagesrcset')).find(src => src == lcpUrl);
```

Comment:
WDYT about adding an optional chaining operator as an extra check?

```suggestion
            src = splitSrcSet(link.imagesrcset || link.getAttribute('imagesrcset'))?.find(src => src == lcpUrl);
```

Reply:
SGTM, I was debating whether that was necessary

Reply:
Technically, it shouldn't be!